newswires/client/src/App.tsx

```diff
@@ -238,23 +255,49 @@ export function App() {
 										margin-left: 8px;
 									`}
 								>
-									<EuiButton
-										size="s"
-										iconType={'popout'}
-										onClick={() =>
-											window.open(
-												configToUrl({
-													...config,
-													view: 'feed',
-													itemId: undefined,
-												}),
-												'_blank',
-												'popout=true,width=400,height=800,top=200,location=no,menubar=no,toolbar=no',
-											)
-										}
-									>
-										New ticker
-									</EuiButton>
+									<EuiShowFor sizes={['xs', 's']}>
+										<Tooltip
+											tooltipContent={'Open ticker in a new window'}
```

Comment:
The tooltip for the ticker button in the search summary says `Open new ticker`. It would be nice to make both messages consistent.